app/src/main/java/dev/patrickgold/florisboard/settings/AboutActivity.kt

```diff
@@ -42,8 +49,22 @@ class AboutActivity : AppCompatActivity() {
         val toolbar = findViewById<Toolbar>(R.id.toolbar)
         setSupportActionBar(toolbar)
 
-        // Set app version string
-        binding.appVersion.text = "v" + AppVersionUtils.getRawVersionName(this)
+        // Set app version string and onClickListener
+        val appVersion = "v" + AppVersionUtils.getRawVersionName(this)
+        binding.appVersion.text = appVersion
+        binding.appVersion.setOnClickListener {
+            val florisBoard = FlorisBoard.getInstance()
+            val isImeSelected = checkIfImeIsSelected(this)
+            if (isImeSelected) {
+                FlorisClipboardManager.getInstance().addNewPlaintext(appVersion)
+                Toast.makeText(florisBoard, R.string.about__version_copied__title, Toast.LENGTH_LONG).show()
```

Comment:
The context should be `this`.
```suggestion
                Toast.makeText(this, R.string.about__version_copied__title, Toast.LENGTH_LONG).show()
```